pkg/controller/cronjob/cronjob_controllerv2_test.go

```diff
@@ -322,230 +323,156 @@ func (f *fakeQueue) AddAfter(key interface{}, t time.Duration) {
 }
 
 // this test will take around 61 seconds to complete
-func TestController2_updateCronJob(t *testing.T) {
-	cjc := &fakeCJControl{}
-	jc := &fakeJobControl{}
-	type fields struct {
-		queue          *fakeQueue
-		recorder       record.EventRecorder
-		jobControl     jobControlInterface
-		cronJobControl cjControlInterface
-	}
-	type args struct {
-		oldJobTemplate *batchv1beta1.JobTemplateSpec
-		newJobTemplate *batchv1beta1.JobTemplateSpec
-		oldJobSchedule string
-		newJobSchedule string
-	}
+func TestControllerV2UpdateCronJob(t *testing.T) {
 	tests := []struct {
 		name              string
-		fields            fields
-		args              args
+		oldCronJob        *batchv1beta1.CronJob
+		newCronJob        *batchv1beta1.CronJob
 		deltaTimeForQueue time.Duration
```

Comment:
Maybe this can be named to `expectedDeltaTimeForRequeue` for readibility?

Reply:
Good point will update once the v1 bits are in.